lib/licenses.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const licenses = {
+  'permissive': {
+    licenses: ['MIT', 'Apache-2.0', 'BSD-3-Clause', 'BSD-2-Clause'],
```

Comment:
Are these SPDX IDs or are they github-specific?

Reply:
These are SPDX ids. Rename `licenses` to `spxdLicenseIds`?